lib/l10n/intl_it.arb

```
@@ -55,5 +55,144 @@
+  "createdOn": "Creato il",
+  "paymentMethodsLabel": "Metodi di pagamento",
+  "cancelTrade": "Annulla Scambio",
+  "areYouSureCancel": "Sei sicuro di voler annullare questo scambio?",
+  "cooperativeCancelMessage": "Se confermi, inizierai un annullamento cooperativo con la tua controparte.",
+  "acceptCancelMessage": "Se confermi, accetterai l'annullamento cooperativo iniziato dalla tua controparte.",
+  "confirm": "Conferma",
+  "buy": "COMPRA",
+  "sell": "VENDI",
+  "buying": "COMPRANDO",
+  "selling": "VENDENDO",
+  "creatingNewOrder": "CREAZIONE NUOVO ORDINE",
```

RuboCop (1.75.5)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)